graftm/pplacer.py

```diff
@@ -3,12 +3,22 @@
 import json
 import timeit
 import logging
+import time
 
 from Bio import SeqIO
 
 from graftm.classify import Classify
 from graftm.housekeeping import HouseKeeping
 
+def timeit(method):
```

Comment:
I don't know, does using this slow down the code significantly? An alternative might just be to print out timestamps